cmd/updater/update.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 # shellcheck disable=2009,2093,2164
 
+MIN_VERSION="3.8.0"
```

Comment:
I like the addition of this variable, but maybe it could be a little misleading to what it's referring to, maybe `UPDATER_MIN_VERSION` ?

Reply:
updated